src/AspNet.Security.OAuth.WordPress/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+    "version": "1.0.0-alpha1-*",
+    "description": "ASP.NET 5 security middleware enabling WordPress authentication.",
+    "authors": [ "Jerrie Pelser" ],
+    "owners": [ "Kévin Chalet", "Jerrie Pelser" ],
+
+    "projectUrl": "https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers",
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0.html",
+
+    "dependencies": {
+        "AspNet.Security.OAuth.Extensions": { "type": "build", "version": "1.0.0-*" },
+        "Microsoft.AspNet.Authentication.OAuth": "1.0.0-*",
+        "Microsoft.Framework.NotNullAttribute.Internal": { "type": "build", "version": "1.0.0-*" }
+    },
+
+    "frameworks": {
+        "dnx451": { },
+        "dnxcore50": { }
```

Comment:
There's a bug with dependency resolution in the latest DNX. Could you please add this workaround to make CI happy? :thought_balloon: 

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dev/src/AspNet.Security.OAuth.GitHub/project.json#L27-L31

Thanks!